README.md

```diff
@@ -48,20 +48,21 @@ declaring [SwiftNIO's Public API](https://github.com/apple/swift-nio/blob/main/d
 
 ### `swift-nio-transport-services ` 1.x
 
-`swift-nio-transport-services` versions 1.x is part of the SwiftNIO 2 family of repositories and does not have any dependencies besides [`swift-nio`](https://github.com/apple/swift-nio), Swift 5.5.2, and an Apple OS supporting `Network.framework`. As the latest version, it lives on the [`main`](https://github.com/apple/swift-nio-transport-services) branch.
+`swift-nio-transport-services` versions 1.x is part of the SwiftNIO 2 family of repositories and does not have any dependencies besides [`swift-nio`](https://github.com/apple/swift-nio), Swift 5.6, and an Apple OS supporting `Network.framework`. As the latest version, it lives on the [`main`](https://github.com/apple/swift-nio-transport-services) branch.
 
 To depend on `swift-nio-transport-services `, put the following in the `dependencies` of your `Package.swift`:
 
     .package(url: "https://github.com/apple/swift-nio-transport-services.git", from: "1.0.0"),
 
-The most recent versions of SwiftNIO Transport Services support Swift 5.5.2 and newer. The minimum Swift version supported by SwiftNIO Transport Services releases are detailed below:
+The most recent versions of SwiftNIO Transport Services support Swift 5.6 and newer. The minimum Swift version supported by SwiftNIO Transport Services releases are detailed below:
 
 SwiftNIO Extras     | Minimum Swift Version
 --------------------|----------------------
 `1.0.0 ..< 1.11.0`  | 5.0
 `1.11.0 ..< 1.12.0` | 5.2
 `1.12.0 ..< 1.15.0` | 5.4
-`1.15.0 ...`        | 5.5.2
+`1.15.0 ..< 1.16.0` | 5.5.2
```

Comment:
This is not _quite_ right: 1.16.0 does support 5.5.2. I struggled with this a bit myself in my other PRs and eventually put the equivalent of 1.16.1 in this spot.

Reply:
https://github.com/apple/swift-nio-transport-services/pull/172/commits/f1febb1934c0c945d2f9ed82dfff82d041e84347